lib/spacex/models.rb

```diff
@@ -0,0 +1,5 @@
+module SPACEX
```

Comment:
Why are you calling this file `models.rb` but the namespace is actually `capsules` ?

Reply:
If I am remembering my thinking from a year ago correctly, I was thinking that models.rb will autoload all the different models. Currently only loading capsules but as we expand the v4 API, we will add the other models in this file. Similar to how capsules is loaded below on line 3